src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/packageScanner.hpp

```diff
@@ -65,6 +67,11 @@ class PackageScanner final : public AbstractHandler<std::shared_ptr<ScanContext>
                 std::string versionStringLessThan {version->lessThan() ? version->lessThan()->str() : ""};
                 std::string versionStringLessThanOrEqual {version->lessThanOrEqual() ? version->lessThanOrEqual()->str()
                                                                                      : ""};
+                VersionObjectType objectType = VersionObjectType::Unspecified;
+                if (m_mapVendors.find(data->packageFormat()) != m_mapVendors.end())
+                {
+                    objectType = m_mapVendors.at(data->packageFormat());
+                }
```

Comment:
Change this to avoid double implicit find.
```suggestion
                VersionObjectType objectType = VersionObjectType::Unspecified;
                const auto it = m_mapVendors.find(data->packageFormat());
                if (it != m_mapVendors.end())
                {
                    objectType = it->second;
                }
```

Reply:
Done! Thanks!